src/filterdb.py

```diff
@@ -0,0 +1,82 @@
+import csv
```

Comment:
all files in the src folder to be moved to the browse/scripts folder and the src folder to be removed.

Reply:
fixed